helm/frost-server/values.yaml

```diff
@@ -29,8 +29,8 @@ frost:
       path: /(.*)
       annotations:
       tls:
-        enabled: false
-        secretName:
+        enabled: true
```

Comment:
I think it's better to leave the `frost.http.ingress.tls.enabled` default value to false, like before. So it's working in every cluster, even if no tls is configured.